docs/try-it-in-the-playground.md

```diff
@@ -30,6 +30,12 @@ using the built-in response printer:
 sdk.listings.query({per_page: 5}).then(printResponse);
 ```
 
+Alternatively you can use the pr() helper function:
```

Comment:
Comment: The `pr` helper is neat, so it could be also advertised as the primary pretty printing method, where as the `.then(printResponse)` could be marketed as the "alternative" version.

Reply:
I see your point but decided to not act on this. I feel it's a coin toss which one of the options is more useful. Also, the order in which we present the information will likely make very little difference in terms of discoverability and usage.